gnomad_constraint/utils/constraint_basics.py

```diff
@@ -81,3 +93,64 @@ def prepare_ht_for_constraint_calculations(ht: hl.Table):
     # Filter out locus with undefined exome coverage
     ht = ht.filter(hl.is_defined(ht.exome_coverage))
     return ht
+
+
+def create_constraint_training_dataset(
+    exome_ht: hl.Table,
+    context_ht: hl.Table,
+    mutation_ht: hl.Table,
+    max_af: float = 0.001,
+    kept_annotations: Tuple[str] = (
+        "context",
+        "ref",
+        "alt",
+        "methylation_level",
+        "exome_coverage",
+    ),
+    pops: Tuple[str] = POPS,
+    grouping: Tuple[str] = ("exome_coverage",),
+    partition_hint: int = 100,
+) -> hl.Table:
+    """
+    Count the observed variants and possible variants by exome coverage.
+
+    :param exome_ht: Preprocessed exome Table.
+    :param context_ht: Preprocessed context Table.
+    :param mutation_ht: Preprocessed mutation rate Table.
+    :param max_af: Variants with AF above than AF cutoff will be removed, defaults to 0.001.
+    :param kept_annotations: Annotations to keep in the context Table.
+    :param pops: List of populations. Defaults to `POPS`.
```

Comment:
```suggestion
    :param pops: List of populations. Default is `POPS`.
```

Also, List of populations for...